<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RMSDPropertyDesigner</class>
 <widget class="QWidget" name="RMSDPropertyDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RMSD Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="generalGB">
         <property name="title">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="structuresGB">
         <property name="title">
          <string>Structures</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Target:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QObjectLinkWidget" name="targetLO" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Number of atom couples:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="mapCount1LE">
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Template:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QObjectLinkWidget" name="templateLO" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="propFlagsGB">
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="allowOnlySameZCouplesCB">
            <property name="text">
             <string>Only atoms with the same Z couples</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="massWeightedCB">
            <property name="text">
             <string>Mass weighted</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="doNotFitCB">
            <property name="text">
             <string>Do not fit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dataGB">
         <property name="title">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox1">
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mapTab">
      <attribute name="title">
       <string>Map</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeView" name="mapTV">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of atom couples:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mapCount2LE">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="removeSelectedCouplePB">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeHydrogenCouplesPB">
            <property name="text">
             <string>Remove H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeAllCouplePB">
            <property name="text">
             <string>Remove all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Manual mapping</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="startManualMappingPB">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopManualMappingPB">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Automatic mapping</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QPushButton" name="mappingByDistancePB">
            <property name="text">
             <string>by distance</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QQuantitySpinBox" name="distanceTresholdSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="mappingBySerialIndexPB">
            <property name="text">
             <string>by serial index</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolsTab">
      <attribute name="title">
       <string>Tools</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="alignPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Align</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideMappedAtomPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Hide mapped atoms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideUnmappedAtomsPB">
         <property name="text">
          <string>Hide unmapped atoms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showAllAtomsPB">
         <property name="text">
          <string>Show all atoms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="templateToTargetPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Template atom positions -&gt; Target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="targetToTemplatePB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Target atom positions -&gt; Template</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="imprintTemplateTopologyToTargetPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Imprint template topology to target</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="imprintTargetTopologyToTemplatePB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Imprint target topology to template</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>300</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphicsTab">
      <attribute name="title">
       <string>Graphics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="graphicsGB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Graphics link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="graphFlagsGB">
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="visibleCB">
            <property name="text">
             <string>Visible</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="highlightedCB">
            <property name="text">
             <string>Highlighted</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox2">
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="refByTab">
      <attribute name="title">
       <string>RefBy</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QObjectLinkWidget</class>
   <extends>QWidget</extends>
   <header location="global">ObjectLinkWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQuantitySpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">QuantitySpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
